pkg/console/starter/starter.go

```diff
@@ -103,13 +106,35 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		oauthinformers.WithTweakListOptions(tweakListOptionsForOAuth),
 	)
 
-	recorder := ctx.EventRecorder
+	resourceSyncerInformers := v1helpers.NewKubeInformersForNamespaces(
+		kubeClient,
+		api.OpenshiftConfigManagedNamespace,
+	)
 
 	operatorClient := &operatorclient.OperatorClient{
 		Informers: operatorConfigInformers,
 		Client:    operatorConfigClient.OperatorV1(),
 	}
 
+	resourceSyncer := resourcesynccontroller.NewResourceSyncController(
+		operatorClient,
+		resourceSyncerInformers,
+		v1helpers.CachedSecretGetter(kubeClient.CoreV1(), resourceSyncerInformers),
```

Comment:
It would be nice if the `ConfigMapGetter` and `SecretGetter` were optional, it would eliminate our extra RBAC rules.

Reply:
No longer using resource sync.